frontend/packages/kubevirt-plugin/integration-tests/tests/utils/scripts/expect-vm-env-readable.sh

```diff
@@ -21,7 +21,7 @@ spawn virtctl console $vm_name -n $vm_namespace --timeout 7
 
 send -h "\n"
 
-sleep 3
+sleep 60
 
 send -h \004
```

Comment:
Please add a comment here saying we are sending `ctrl+D` to logout any logged-in users .

Reply:
Ops, forgot to add that, fixed.

Reply:
+1